apps/metadata-editor-e2e/src/e2e/edit.cy.ts

```diff
@@ -215,7 +218,59 @@ describe('editor form', () => {
             .should('be.visible')
         })
       })
+
+      describe('image field', () => {
+        beforeEach(() => {
+          // Delete existing image if it exists
+          cy.get('gn-ui-image-input').then(($input) => {
```

Comment:
I think this one could be simpler? (no `.then()` needed)

Reply:
I don't see how delete an optional image without a ".then." at this point we are not sure the image is on the gn-ui-image-input, so make a delete directly make the beforeEach fails.

Reply:
+1 with @Guillaume-d-o, I don't think it is possible to remove `then` because cypress will throw an error when the image is gone before the second test. It's not an issue to keep it IMO.
If we want to simplify a bit it could be : 

```suggestion
  cy.get('gn-ui-image-input img').then(($img) => {
    if ($img.length > 0) {
      cy.get('[data-cy="delete-image"]').click();
      cy.editor_wrapPreviousDraft();
      cy.editor_publishAndReload();
      cy.get('gn-ui-image-input img').should('not.exist');
    }
  });
```